File: 백준/Bronze/1267. 핸드폰 요금/핸드폰 요금.py
n = int(input())

times = list(map(int, input().split()))

total_y = 0
total_m = 0

for time in times:
    cost_y = ((time // 30) + 1) * 10
    total_y += cost_y
    
    cost_m = ((time // 60) + 1) * 15
    total_m += cost_m
    
if total_y < total_m:
    print(f"Y {total_y}")
elif total_m < total_y:
    print(f"M {total_m}")
else:
    print(f"Y M {total_y}")

File: 백준/Bronze/1267. 핸드폰 요금/핸드폰 요금.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        
        int totalY = 0;
        int totalM = 0;
        
        for (int i =0; i < n; i++) {
            int time = sc.nextInt();
            
            int costY = ((time / 30) + 1) * 10;
            totalY += costY;
            
            int costM = ((time / 60) + 1) * 15;
            totalM += costM;
        }
        sc.close();
        
        if (totalY < totalM)
            System.out.println("Y " + totalY);
        else if (totalM < totalY)
            System.out.println("M " + totalM);
        else
            System.out.println("Y M " + totalY);
    }
}